sound/soc/sof/sof-priv.h

```diff
@@ -308,6 +308,11 @@ struct snd_sof_pcm_stream {
 	struct snd_pcm_substream *substream;
 	struct work_struct period_elapsed_work;
 	bool d0i3_compatible; /* DSP can be in D0I3 when this pcm is opened */
+	/*
+	 * flag to indicate that the DSP pipelines should be kept
+	 * active or not on the stream suspended
```

Comment:
flag to indicate if the PCM streams should be in SUSPENDED or RUNNING states after a power state transition (device D0i3/D3, system S0/S3 suspend)

Reply:
Ditto.

Reply:
"on the stream suspended" is not quite right (meaning and grammar)

flag to indicate that the DSP pipelines should be kept active or not when the PCM stream becomes SUSPENDED